library/src/scripts/layout/frame/frameStyles.ts

```diff
@@ -85,7 +85,7 @@ export const frameClasses = useThemeCache(() => {
         minHeight: 0, // https://bugs.chromium.org/p/chromium/issues/detail?id=927066
         $nest: {
             [`.${bodyWrap}`]: {
-                flex: "1 0 auto",
+                flexGrow: 1,
```

Comment:
```suggestion
                flexGrow: 1,
```

I'm particularly worried about this change.

[I _just_ had to to change it a couple days ago](https://github.com/vanilla/vanilla/pull/9663) to fix scrolling in modals in IE11. Did you test it out in browserstack?

Reply:
I tested myself, and it didn't cause a regression with that other thing.